code/game/machinery/iv_drip.dm

```diff
@@ -145,6 +145,8 @@
 				update_icon()
 
 /obj/machinery/iv_drip/attack_hand(mob/user)
+	if (!ishuman(user))
```

Comment:
Iirc, the consensus was that silicons (except the AI since the issue has obviously been accepted) could attach/detach IVs, when i did my IV bugfixing PR.